RecoLocalCalo/HGCalRecProducers/plugins/HeterogeneousHGCalHEFCellPositionsConditions.cc

```diff
@@ -1,3 +1,4 @@
+#include "RecoLocalCalo/HGCalRecProducers/plugins/KernelManagerHGCalCellPositions.h"
```

Comment:
```suggestion
#include "KernelManagerHGCalCellPositions.h"
```
to rely more clearly on locality of the file